package.json

```diff
@@ -47,8 +48,10 @@
     "eslint-plugin-react": "^7.11.1",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^23.6.0",
+    "jquery": "^3.3.1",
```

Comment:
are you going to use this?

Reply:
@Kachulio1  I have removed it together with Popper.js since I will not be using the Bootstrap plugins anymore. Will now only use the Bootstrap CSS file.